website/data/plugin-testing-nav-data.json

```diff
@@ -0,0 +1,39 @@
+[
+  { "heading": "Testing" },
+  { "title": "Overview", "path": "" },
+  { "title": "Migrate to the Testing Module", "path": "migrating" },
+  {
+    "title": "Acceptance Testing",
+    "routes": [
+      {
+        "title": "Overview",
+        "path": "acceptance-tests"
+      },
+      {
+        "title": "Test Cases",
+        "path": "acceptance-tests/testcase"
+      },
+      {
+        "title": "Test Steps",
+        "path": "acceptance-tests/teststep"
+      },
+      {
+        "title": "Sweepers",
+        "path": "acceptance-tests/sweepers"
+      }
+    ]
+  },
+  {
+    "title": "Testing API",
+    "path": "testing-api",
+    "hidden": true
+  },
```

Comment:
I think we should just delete this empty page and remove from this navigation file.

```suggestion
```